lookout/style/typos/benchmarks/typo_commits_report.py

```diff
@@ -127,11 +165,25 @@ def _finalize(self, reports: Iterable[Dict[str, str]]) -> Iterator[Dict[str, str
         :param reports: Reports generated by `TypoCommitsReporter.generate_commit_dataset_report()`
         :return: Summarized final report
         """
-        # TODO(zurk): Add a report template file with metric we want to calculate.
-        sum_metric = self.get_metrics_billet()
+        scores = self.get_metrics_billet()
```

Comment:
billet? 🤔 there is no such word. Do you mean ticket?

Reply:
https://translate.google.com/#view=home&op=translate&sl=auto&tl=ru&text=billet

Reply:
ok, nobody uses this word usually. "stub" is more common